docs/migration/manual-migration.mdx

```diff
@@ -58,6 +58,89 @@ mechanics from Remix loaders and hooks:
   transfered to `useTrackPage` and `useProductPage` hooks
 - Loading state management is done directly at route loader level
 
+#### `withTrackOnMount` HOC depreciation
+
+`withTrackOnMount` has been deprecated in favor of a new hook `useTrackOnMount`.
+The new hook has a slightly different signature:
+
+- `isResolvedFromProps`, previously a function, parameter has been replaced with
+  `hasResolved` which is now a boolean
+- `mapPropsToProperties`, previously a function, has been renamed to
+  `createProperties` but is otherwise unchanged
+- `shouldMountEvent`, `shouldUpdateEvent` and `trackEvent` have been left
+  unchanged.
```

Comment:
I'm curious how a table would look, for example:



| old prop             | new prop         | prop type  |
|----------------------|------------------|------------|
| isResolvedFromProps  | hasResolved      | `boolean`  |
| mapPropsToProperties | createProperties | `function` |

`shouldUpdateEvent` and `trackEvent` have been left
  unchanged.

Reply:
Indeed, should look good. I'll give it a try.

Reply:
Wait... This change shouldn't be there. It has been done in https://github.com/front-commerce/developers.front-commerce.com/commit/44e18f9df7c6a66ccb5515556a46a3209053b2eb, which is on `next already`.

Reply:
Rebased on next, change is gone indeed.